### ISSUE TITLE
[version:0.22.2]Warning: Failed prop type: Invalid prop `modalTitle` of type `Object` supplied to `BottomModal`, expected instance of `Element`.

### ISSUE THREAD

**Issue by xkungfu** [jacklam718/react-native-modals#251]
react-native-modals 0.22.2
start by: expo start web

**Warning: Failed prop type: Invalid prop `modalTitle` of type `Object` supplied to `BottomModal`, expected instance of `Element`.**

full warning:

```

index.js:1 Warning: Failed prop type: Invalid prop `modalTitle` of type `Object` supplied to `BottomModal`, expected instance of `Element`.
    in BottomModal (at FooterButtons.tsx:275)
    in FooterButtons (at GiftedChat.tsx:1477)
    in div (created by View)
    in View (at GiftedChat.tsx:1472)
    in div (created by View)
    in View (at ActionSheet/index.tsx:68)
    in ActionSheet (at ActionSheetProvider.tsx:30)
    in ActionSheetProvider (at GiftedChat.tsx:1469)
    in div (created by View)
    in View (created by SafeAreaView)
    in SafeAreaView (at GiftedChat.tsx:1468)
    in GiftedChat (at App.tsx:610)
    in div (created by View)
    in View (at App.tsx:603)
    in App (at withExpoRoot.web.js:13)
    in RootErrorBoundary (at withExpoRoot.web.js:12)
    in ExpoRootComponent (at registerRootComponent.web.js:6)
    in RootComponent
    in div (created by View)
    in View (created by AppContainer)
    in div (created by View)
    in View (created by AppContainer)
    in AppContainer
```

my code:

```

        <ModalPortal />
        <BottomModal
          visible={this.state.bottomModalAndTitle}
          onTouchOutside={() => this.setState({ bottomModalAndTitle: false })}
          height={0.2}
          width={1}
          onSwipeOut={() => this.setState({ bottomModalAndTitle: false })}
          modalTitle={
            <ModalTitle
              style={{
                backgroundColor: 'red',
              }}
              title="Bottom Modal"
              hasTitleBar
            />
          }
        >
          <ModalContent
            style={{
              flex: 1,
              backgroundColor: 'fff',
              position: 'relative',
            }}
          >
            <TouchableOpacity
              style={{
                position: 'absolute',
                right: 15,
                top: -40,
              }}
              onPress={() => this.closeDialog()}
            >
              <Text>CLOSE</Text>
            </TouchableOpacity>
            <Text>
              Bottom Modal with Title
            </Text>
          </ModalContent>
        </BottomModal>

```

**Comment by xkungfu** [comment]
same error in example: https://github.com/jacklam718/react-native-modals/tree/master/example

build by expo: expo start web

```


index.js:1 Warning: Failed prop type: Invalid prop `modalTitle` of type `Object` supplied to `BottomModal`, expected instance of `Element`.
    in BottomModal (at DemoScreen.js:322)
    in App (created by SceneView)
    in StaticContainer
    in StaticContainer (created by SceneView)
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by CardContainer)
    in div (created by View)
    in View (created by CardContainer)
    in div (created by View)
    in View (created by CardContainer)
    in div (created by View)
    in View (created by ForwardRef(CardSheet))
    in ForwardRef(CardSheet) (created by Card)
    in div (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent (created by ForwardRef(AnimatedComponentWrapper))
    in ForwardRef(AnimatedComponentWrapper) (created by Card)
    in Dummy (created by Card)
    in div (created by View)
    in View (created by AnimatedComponent)
    in AnimatedComponent (created by ForwardRef(AnimatedComponentWrapper))
    in ForwardRef(AnimatedComponentWrapper) (created by Card)
    in div (created by View)
    in View (created by Card)
    in Card (created by CardContainer)
    in CardContainer (created by CardStack)
    in div (created by View)
    in View (created by MaybeScreen)
    in MaybeScreen (created by CardStack)
    in div (created by View)
    in View (created by MaybeScreenContainer)
    in MaybeScreenContainer (created by CardStack)
    in CardStack (created by Context.Consumer)
    in KeyboardManager (created by Context.Consumer)
    in div (created by View)
    in View (created by NativeSafeAreaView)
    in NativeSafeAreaView (created by SafeAreaProvider)
    in SafeAreaProvider (created by Context.Consumer)
    in SafeAreaProviderCompat (created by StackView)
    in div (created by View)
    in View (created by StackView)
    in StackView (created by StackNavigator)
    in StackNavigator (at Navigator.js:12)
    in EnsureSingleNavigator (created by ForwardRef(BaseNavigationContainer))
    in ForwardRef(BaseNavigationContainer) (created by ForwardRef(NavigationContainer))
    in ThemeProvider (created by ForwardRef(NavigationContainer))
    in ForwardRef(NavigationContainer) (at Navigator.js:11)
    in Navigator (at App.js:4)
    in Root (created by ExpoRootComponent)
    in ExpoRootComponent (created by RootComponent)
    in RootComponent
    in div (created by View)
    in View (created by AppContainer)
    in div (created by View)
    in View (created by AppContainer)
    in AppContainer
```

**Comment by jacklam718** [comment]
should be fixed in `v0.22.3`

**Comment by xkungfu** [comment]
very nice.thanks.
